### DISCUSSION TITLE
Are scheduled actions disabled in .github repos?

### DISCUSSION THREAD

**Discussion by thclark** [community/community#13751] [Actions] [answered]
I recently got this email:
```
Scheduled workflows are disabled automatically after 60 days of repository inactivity. [Learn more.](https://docs.github.com/actions/managing-workflow-runs/disabling-and-enabling-a-workflow)
```
I get it, on a normal code repository, because there's no point repeating actions over and over on no changes. 

**BUT** I have actions in the `.github` repository (which sits there for months at a time unchanged)... and those actions act on other repos daily or weekly (e.g. making sure labels are consistent, checking we're using our latest devcontainer stack, sorting issues on the board, etc etc).

Do I have to touch my `.github` repo every two months just to keep my maintenance actions running?

**Comment by ethomson** [comment, marked as answer]
> Do I have to touch my .github repo every two months just to keep my maintenance actions running?

Yes, I'm afraid that you do. There's really know way for us to tell if your workflow is something that only has automation that you've set up once and don't _need_ to touch (but want to keep), versus something that has gotten old and stale and forgotten about, versus testing you did that is just going to run up your bill.

(And we don't want to charge you for something that you've forgotten about or that isn't bringing you any value! That's the sort of thing that makes everybody unhappy.)

So, yes, we do require some regular activity in the repository so that we know that you want us to keep running workflows. I apologize if that's an inconvenience.

We can look at some less intrusive ways to keep this going in the future, and I'll put that on the backlog.

**Comment by thclark** [comment]
Thanks for the clarification @ethomson :)
As a workaround, one could make a scheduled action to add a comment on an issue every month in that repository ;)
